ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/YesNoMapper.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+@Component
```

Comment:
This mapper is overkill. Not needed

Reply:
The null check you are suggesting to encapsulate can very well be within the Enumeration. It will be like another method like valueOf. To do just Enum to Enum we dont need a Component to be declared. But if its preferred, No objection.